### ISSUE TITLE
Accessibility: Content not included in landmarks

### ISSUE THREAD

**Issue by banksashleym** [JacksonCollege/jc-theme-2015#265]
**Description of this issue:**
HTML5 or WAI-ARIA landmarks are used on the page, but not all content is included.

When using HTML5 or WAI-ARIA landmarks it is best practice to include all content on the page in landmarks. In this way users of assistive technologies can use the landmarks for navigating the page without losing track of content.

**How to fix it:**
Make sure that all content on the page is included in HTML5 or WAI-ARIA landmarks.

Seems like this may be coming from Google Translate and the Google Tag Manager? Google translate is giving quite a few errors and I'm not sure if there is anything we can do about that? Are there any alternatives to google translate that would maybe be more accessible? Thanks!

The scan shows all pages are affected by this error.

![2](https://user-images.githubusercontent.com/15264315/46360841-47372800-c63a-11e8-8af9-4bc2ba00e527.PNG)

**Comment by d4mation** [comment]
This change is now on Staging. 

The Google Tag Manager fix was pretty simple as it is not an element meant to ever be visible, so I just hid it from accessibility devices. 

For Google Translate, it is a little more tricky. I have it set to toggle its visibility state based on whether or not that panel is open (It shows when you have it set to translate the site and then click on text) but when it is open, the original problem exists again.

I gave the element the `form` Aria Role which I don't know if it is fully appropriate, but it does make the accessibility issue go away. https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Form_Role

This may be something that an accessibility expert should look at to find the best solution.

---

There is this plugin which could be used instead. https://wordpress.org/plugins/google-language-translator/

However, I just tested it and it unfortunately has the same problem. I am unsure of any other solutions that will allow of an in-site instant translation like what Google Translate offers.
